src/Commands/MakeCommand.php

```diff
@@ -122,11 +129,29 @@ protected function createTest($force = false)
         return $testPath;
     }
 
+    protected function createPest($force = false)
```

Comment:
I'm OK with this, but I think it would be better to add a flag as a second param to `createTest()` because there is only one small difference between these two methods. I think abstracting the change is the best way here.

Reply:
Ah I see there is also a difference in calling `->pestContests()` instead of `testContents`. I'm more OK with this then.